src/lib/containers/download_list_v2/DownloadListStats.tsx

```diff
@@ -36,7 +36,7 @@ export default function DownloadListStats() {
         </div>
       )}
       {isFetching && (
-        <div className="EntityFinderDetailsView__Placeholder">
+        <div className="placeholder">
```

Comment:
Was reusing a style defined for the EntityFinderDetailsView.  Pulled this out into a mixin, and use it in a general "placeholder" css class.